package.json

```diff
@@ -23,14 +23,19 @@
     "css-check": "ts-node --esm ./scripts/css-check.ts"
   },
   "dependencies": {
-    "@preact/signals-react": "^1.3.6",
+    "yup": ">=1.3.2",
+    "@preact/signals-react": "^1.3.6"
```

Comment:
I still think it makes sense here to keep yup and the signals library as direct dependencies as the user may not have it installed